tools/explorer/tt_adapter/src/tt_adapter/mlir.py

```diff
@@ -522,7 +522,12 @@ def build_graph(module):
                     op_to_graph_node[op] = graph_node
 
                     for operand in op.operands:
-                        if isinstance(operand, ir.Value):
+                        if isinstance(operand, ir.Value) and isinstance(
```

Comment:
Should we check that owner is not Op rather than it's a block maybe?

Reply:
yeah that sounds good. I'll add that in